CHANGELOG.md

```diff
@@ -63,6 +63,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-3319**
+  - Removed @cumulus/api/models/schema and changed all references to
+    @cumulus/api/lib/schema in docs and related models
+  - Removed @cumulus/api/models/errors.js
```

Comment:
You also removed `packages/api/models/schemas.js`, though maybe it's not as important to note since `packages/api/lib/schemas.js` is its replacement? 

```suggestion
  - Removed @cumulus/api/models/errors.js
  - Removed @cumulus/api/models/schemas.js
```

Reply:
@botanical  I think I noted it on L67, although the .js is probably a mistake in L69.      🤔